Automattic/newspack-ads
# PHPCS Fixes

This PR replaces the `.dist` versions of `phpcs.xml` and `phpunit.xml`. Accidentally this also includes a minor fix to the formatting of Ad Unit titles, to properly format single and double quotes in Ad Unit names.

To Test:

- Run `composer update`
- Verify that PHPCS warnings and errors are working.
- Verify that tests still run successfully.
